checkups/kubevirt-vm-latency/cmd/main.go

```diff
@@ -23,14 +23,16 @@ import (
 	"log"
 	"os"
 
+	"github.com/kiagnose/kiagnose/kiagnose/environment"
```

Comment:
Why is there another `kiagnose` level?
```suggestion
	"github.com/kiagnose/kiagnose/environment"
```

Reply:
Please ignore this comment.
Its the <org>/<repo name>
My bad.